AbstractClient.php

```diff
@@ -91,36 +103,41 @@ protected function performRequest(Endpoint\EndpointInterface $endpoint)
         return $body;
     }
 
+    protected function getUriFromEndpoint(EndpointInterface $endpoint): string
```

Comment:
I would rename this to uriWithPrependPath. It says much more wath the function is doing.

Reply:
Done!